x-pack/plugins/observability_shared/public/components/section/section.tsx

```diff
@@ -62,24 +60,12 @@ export const Section = styled.div`
 `;
 
 export type SectionLinkProps = EuiListGroupItemProps;
-export function SectionLink({
-  showNewBadge,
```

Comment:
I'd probably keep this in case in the future we want to add it again on another tab.

Reply:
As it wasn't used anymore I decided to remove it so it's not confusing why it's there when it's not used anywhere - I think if we ever need that it can be added so we can avoid having code that it's not used - do you know if we plan to add new features where this badge will be used in the near future? If so I can revert that.

Reply:
Hard to say. But it's up to you. This is not a big deal to keep this code as well.

Reply:
Ok, let's leave the change and we can add it in case it's needed again